priv/repo/migrations/20160802030642_create_note.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Notox.Repo.Migrations.CreateNote do
+  use Ecto.Migration
+
+  def change do
+    create table(:notes) do
+      add :title, :string
+      add :description, :string
+
+      timestamps
```

Comment:
Once you fix this stuff you'll have to rollback the database with `mix ecto.rollback`. That will roll back a _single_ migration, whereas `mix ecto.migrate` migrates all migrations that haven't run yet.